### PR TITLE
CRITICAL LAG FIX FOR SINGULARITIES AND THROWING

### PR DESCRIPTION
**Pull request by Iamgoofball**
fixes INTENSE LAG CAUSED BY SINGULARITIES PULLING BULLSHIT

### CODE REVIEW

**Comment by kevinz000** [comment]
TTThrowing someone in space or in hall without gravity. My sides.

**Comment by Fox-McCloud** [comment]
This may make people spin forever if thrown.

**Comment by Jordie0608** [comment]
Fox is right, thrown people spin forever.

File: code/game/atoms_movable.dm [hunk 1]
```diff
@@ -158,7 +158,8 @@
 	//use a modified version of Bresenham's algorithm to get from the atom's current position to that of the target
 
 	src.throwing = 1
-	SpinAnimation(5, 1)
+	if(!istype(target, /obj/singularity)) // turns out 1000+ spinning objects being thrown at the singularity creates lag - Iamgoofball
```
Comment (Kelenius) [inline code/game/atoms_movable.dm]: Snowflake check - check.

**Comment by Fox-McCloud** [comment]
Please note that it works absolutely fine for objects and that they do follow the behavior you intend.

**Comment by Iamgoofball** [comment]
fucking mobs

File: code/game/atoms_movable.dm (hunk 1 above)
Comment (optimumtact) [inline code/game/atoms_movable.dm]: Yeah this is pretty awful, anyone have a better solution? A second throw proc without spin perhaps?

Comment (RemieRichards) [inline code/game/atoms_movable.dm]: An argument... on throw_at()
Cmon guys, it's that simple.

Comment (optimumtact) [inline code/game/atoms_movable.dm]: disgusting

**Comment by Iamgoofball** [comment]
OKAY INFINITE SPIN BULLSHIT REVERTED

MERGE WHEN

**Comment by Anonus** [comment]
5~ days until merge.

**Pull request merged**